.azure/pipelines/jobs/default-build.yml

```diff
@@ -77,6 +77,9 @@ jobs:
     dependsOn: ${{ parameters.dependsOn }}
     ${{ if ne(parameters.condition, '') }}:
       condition: ${{ parameters.condition }}
+    ${{ if ne(parameters.enableRichCodeNavigation, '') }}:
+      enableRichCodeNavigation: true
+      richCodeNavigationLanguage: 'csharp,typescript,java'
```

Comment:
Curious: Any other supported languages e.g. this repo contains JavaScript, C++, and a bit of F#❔